WcaOnRails/app/views/regulations/scrambles/index.html.erb

```diff
@@ -10,7 +10,7 @@
       <strong><%= link_to latest_jarfilename, "./tnoodle/#{latest_jarfilename}" %></strong>
     </span>
     <br/>
-    Last official change: December 3, 2018
+    Last official change: July 13rd, 2019
```

Comment:
Make sure to update this when it lands!

Reply:
Isn't this updated?

Reply:
I think we should use a date of July 15, 2009. That way the release is officially on Monday.

(It's Monday for me in Melbourne!)

Reply:
Updated to July 14th

Reply:
Ops, merged with July 14th... I saw your comment a bit late on this... Sorry.

Reply:
No worries. If you wouldn't mind, though, could you also create a PR to update this to the 15th? Thanks!